tests/dummydata.py

```diff
@@ -0,0 +1,316 @@
+#code implementing the tests variables of each class
+
+from wifi import subprocess_compat as subprocess
+from wifi.scheme import extract_schemes, Scheme
+from wifi.scan import Cell
+import string
+
+class ScanTesting(object):
```

Comment:
If you are just using this for a namespace, you might as well just use a dictionary.